src/pages/Lend/LoanChannelCategoryControl.vue

```diff
@@ -781,6 +784,9 @@ export default {
 			const loans = loansData?.lend?.loanChannelsById[0]?.loans?.values ?? [];
 			this.helpmeChooseLoans = loans;
 			this.isLoadingHC = false;
+		},
+		resetPagination() {
+			this.$refs.categoryPagination.clickPage(0);
```

Comment:
We probably don't want to call `clickPage`, since it will trigger an analytics click event, and the user didn't click the pagination in this case. Can we change the offset in this component instead? Looks like this component has a `pageChange` method.

Reply:
Nice catch! fixed